pkg/dispatcher/dispatcher.go

```diff
@@ -19,6 +19,9 @@ func NewInput(r *http.Request) (Input, error) {
 	if v := r.FormValue("query"); v != "" {
 		in.Query = v
 	}
+	if v := r.FormValue("match[]"); v != "" {
+		in.Query = v
```

Comment:
Thank you for your PR:)

Instead of overwriting Input#Query, I would like to add it to Input#Matchers.
https://github.com/kobtea/iapetus/blob/v0.1.2/pkg/dispatcher/dispatcher.go#L10-L15

Since `match[]` seems array, it is better to use `[]string` than `string`.
https://prometheus.io/docs/prometheus/latest/querying/api/#finding-series-by-label-matchers

Reply:
Support multiple `match[]` in latest commit.